CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Added pylint `unspecified-encoding` and `missing-function-docstring` and ignored opensearchpy for lints; added new linting per directory option behind feature flag (([#643](https://github.com/opensearch-project/opensearch-py/pull/643)))
```

Comment:
Just say "Added pylint `unspecified-encoding` and `missing-function-docstring`" here, the rest is implementation detail.